### PR TITLE
Add Shadow of the Tomb Raider: DE as Supported.

### PR DESCRIPTION
**Pull request by lucsoft**
Tested with v2.3.2

### CODE REVIEW

**Review by mattmaniak — CHANGES_REQUESTED** [review]
Please regenerate a src/SUMMARY.md with the game title entry.

File: src/game-support/shadow-of-the-tomb-raider-de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Shadow of the Tomb Raider: Definitive Edition
+<!-- script:Aliases [] -->
```
Comment (mattmaniak) [inline src/game-support/shadow-of-the-tomb-raider-de.md:2]: Add an alias:
```
<!-- script:Aliases [
    "Shadow of the Tomb Raider Definitive Edition"
] -->
```

File: src/game-support/shadow-of-the-tomb-raider-de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- Enable DXVK in Bottle Configuration
+- Define Environment Variables:
+  | Key                 | Value     |
+  | ------------------- | --------- |
+  | VKD3D_CONFIG        | dxr11,dxr |
+  | VKD3D_FEATURE_LEVEL | 12_2      |
+> These changes fixed visual glitches and improved performance.
+
+### Heroic Games Launcher
+- Download in Heroic as normal, select whisky and set the custom environment variables.
```
Comment (mattmaniak) [inline src/game-support/shadow-of-the-tomb-raider-de.md:17]: Please add this below line to link a Steam page of the game:
```
{{#template ../templates/steam.md id= 750920}}
```

**Review by mattmaniak — CHANGES_REQUESTED** [review]

**Review by mattmaniak — APPROVED** [review]
Looks ok. Thanks.

**Review by hahayupgit — APPROVED** [review]

**Pull request merged**